ocb3/Cargo.toml

```
@@ -0,0 +1,42 @@
+[package]
+name = "ocb3"
+version = "0.1.0"
+description = """
+Pure Rust implementation of the OCB3
+Authenticated Encryption with Associated Data (AEAD) Cipher
+"""
+authors = ["RustCrypto Developers"]
+edition = "2021"
+license = "Apache-2.0 OR MIT"
+readme = "README.md"
+documentation = "https://docs.rs/ocb3"
+repository = "https://github.com/RustCrypto/AEADs"
+keywords = ["aead", "encryption", "ocb", "ocb3"]
+categories = ["cryptography", "no-std"]
+rust-version = "1.60"
+
+[dependencies]
+aead = { version = "0.5", default-features = false }
+cipher = "0.4"
+ctr = "0.9"
+subtle = { version = "2", default-features = false }
+zeroize = { version = "1", optional = true, default-features = false }
+
+[dev-dependencies]
+aead = { version = "0.5", features = ["dev"], default-features = false }
+aes = { version = "0.8", default-features = false }
+hex-literal = "0.3"
```

With the MSRV bump this can be:

```suggestion
hex-literal = "0.4"
```